src/engine/WebGPUEngine.ts

```
@@ -24,6 +24,8 @@ import { PipelineManager } from './core/PipelineManager';
 import { UniformManager } from './core/UniformManager';
 import { ComputePass } from './rendering/ComputePass';
 import { RenderPass } from './rendering/RenderPass';
+import { Camera3D } from './Camera3D';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Camera3D'. Consider to delete it or rename it.

'Camera3D' is defined here:


(lint/suspicious/noRedeclare)